pallets/living-assets-ownership/src/functions.rs

```diff
@@ -1,22 +1,54 @@
 //! Contains helper and utility functions of the pallet
 use super::*;
-use frame_support::{ensure, sp_runtime::DispatchResult};
+use frame_support::sp_runtime::{
+	traits::{CheckedAdd, One},
+	DispatchResult,
+};
 
 impl<T: Config> Pallet<T> {
-	/// See [Self::create_collection]
-	pub fn do_create_collection(
-		collection_id: T::CollectionId,
-		who: T::AccountId,
-	) -> DispatchResult {
-		ensure!(
-			!OwnerOfCollection::<T>::contains_key(collection_id),
-			Error::<T>::CollectionAlreadyExists
-		);
+	/// Attempts to create a new collection
```

Comment:
I see what you mean by this, but I think it's better like this.
```suggestion
	/// Creates a new collection
```
if it was, let's say, `try_create_collection` this would be a better description